.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
+nodeLinker: node-modules
```

Comment:
Any reason you want to disable yarn PnP?

Reply:
Yarn has a mode that gets rid of the `node_modules` folder. A summary of it is available here: https://yarnpkg.com/features/pnp

The only downside I've found is that there is a [minimal amount of setup](https://yarnpkg.com/getting-started/editor-sdks) required to get Intellisense working. For vscode that setup is available in is repo if you open the project with the workspace file.

Reply:
Aha. Great! Perhaps we should have a "contributing" section that explains this, but I am sure you've already thought about it and haven't gotten to it yet ❤️